quickstarts/microsoft.recoveryservices/recovery-services-create-vault-enable-diagnostics/README.md

```diff
@@ -23,4 +24,14 @@ To know more about Azure Backup Reporting feature refer [this documentation](htt
 
 This template creates new storage account and oms workspace as part of deployment where diagnostic data gets pushed. If you want to use existing storage account and workspace, skip those resources from template and supply appropriate resource ids of existing storage account and oms workspace.
 
+### Deployment steps 
```

Comment:
remove this section - the process/steps need to be the same for all files in the repo